### DISCUSSION TITLE
POS screen currency change

### DISCUSSION THREAD

**Discussion by ibruteforce94** [btcpayserver/btcpayserver#5407] [Ideas / Feature Requests]
Is it possible to implement a setting which would allow changing between EUR or other and SAT/BTC currency on payment screen of POS app?

![image](https://github.com/btcpayserver/btcpayserver/assets/124496636/4b257bbf-13f2-4d07-a667-5e8d11b07d68)

**Comment by dennisreimann** [comment]
It should be, will give it a try :) /cc @dstrukt

**Reply by dstrukt** [reply]
Was wondering how long until this feature would be requested - let's do it!

**Reply by dennisreimann** [reply]
The question is how we want to so it: It could be done in a simple way as either a toggle between the default currency and SATS — or with a list of currencies to toggle between, which the user can define in the backend. Not sure if the latter is really necessary.

However, this would be a keypad-only feature.

**Reply by ibruteforce94** [reply]
> The question is how we want to so it: It could be done in a simple way as either a toggle between the default currency and SATS — or with a list of currencies to toggle between, which the user can define in the backend. Not sure if the latter is really necessary.
> 
> 
> 
> However, this would be a keypad-only feature.

For me personally, all that's needed is a simple toggle between default currency and bitcoin. 

Dropdown menu of admin selected currencies is also ok solution, but I don't find it necessary same as you. There is a possibility to have two keypad POS apps, one for local customers with local currency and one for international. But that is a really really rare scenario I think.

**Reply by dennisreimann** [reply]
Ok, but you'd want this to be BTC instead of SATS? I think the latter would make more sense as its more likely the keypad will be used with smaller amounts. 

We can approach this once #5396 is merged. Let's talk about how in tomorrow's design meeting, @dstrukt.

**Reply by ibruteforce94** [reply]
Yes, you're right, sats are preferred over btc here
